config/application.rb

```diff
@@ -64,7 +64,7 @@ class Application < Rails::Application
     config.assets.compress = true
     config.assets.precompile = ['*.js', '*.css', '*.css.erb', '*.png', '*.jpg', '*.jpeg', '*.gif']
     config.assets.prefix = '/assets'
-    config.relative_url_root = '/opensearch'
+    # config.relative_url_root = '/opensearch'
```

Comment:
can this be deleted instead of commented out?

Reply:
yup